### PR TITLE
Fix cmake install target

### PR DESCRIPTION
**Pull request by bghgary**
Changes from #1325 caused the install files to have extra files. This is a simpler way to do it.

### CODE REVIEW

**Review by CedricGuillemet — APPROVED** [review]

**Comment by CedricGuillemet** [comment]
Why was it not detected by the install test app? how can we improve the test?

**Comment by bghgary** [comment]
> Why was it not detected by the install test app? how can we improve the test?

It's not detected because the layout is proper, but there were extra files being added. I'm not sure how easy it would be to test.

File: Install/Install.cmake [hunk 1]
```diff
@@ -11,23 +11,19 @@ function(install_targets)
...
-function(install_include_for_target)
-    get_target_property(TARGET_INCLUDE_DIRECTORIES ${ARGV0} INCLUDE_DIRECTORIES)
-    foreach(include_directory IN LISTS TARGET_INCLUDE_DIRECTORIES)
-        if(${ARGC} STREQUAL "1")
-            install_include("${include_directory}")
-        else()
-            install_include("${include_directory}/${ARGV1}")
-        endif()
+function(install_include_for_targets)
+    foreach(target IN LISTS ARGN)
```
Comment (CedricGuillemet) [inline Install/Install.cmake:20]: Is that loop necessary?

Comment (bghgary) [reply Install/Install.cmake:20]: No, but it's more consistent with the other functions.

**Pull request merged**